rslib/src/scheduler/fsrs/simulator.rs

```diff
@@ -141,15 +143,26 @@ impl Collection {
             }
         }
         let days_elapsed = self.timing_today().unwrap().days_elapsed as i32;
-        let new_cards = cards
-            .iter()
-            .filter(|c| c.memory_state.is_none() || c.queue == CardQueue::New)
-            .count()
-            + req.deck_size as usize;
+        let new_cards =
+            cards.iter().filter(|c| c.queue == CardQueue::New).count() + req.deck_size as usize;
```

Comment:
The queue of a new card changes when it is buried or suspended. So, we should check the type instead.

```suggestion
            cards.iter().filter(|c| c.ctype == CardType::New).count() + req.deck_size as usize;
```

`is_included_card` needs a similar change.

Reply:
@Luc-Mcgrady what do you think?